src/Components/Details/Details.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import './Details.css'
+import backArrow from './double-left-arrows.svg'
+
+const Details = ({ allMovies, selectedMovieID, hideSelection }) => {
+
+  const movieToDisplay = allMovies.find(movie => {
```

Comment:
Nice work memoizing this, and it has a GREAT name! This makes the return nice and readable. Great job!